statsdata.php

```diff
@@ -274,4 +274,12 @@ public function getConfigFieldsValues()
             'PS_STATSDATA_PLUGINS' => Tools::getValue('PS_STATSDATA_PLUGINS', Configuration::get('PS_STATSDATA_PLUGINS')),
         );
     }
+    
+    /**
+     * Avoid error message in web server log => http://forge.prestashop.com/browse/BOOM-4826
+     * @return null
+     */
+    public function display() {
```

Comment:
This should be 
public function display($file, $template, $cache_id = NULL, $compile_id = NULL)

to be compatible with ModuleCore::display

Reply:
We tried this solution but it was not enough since these errors kept being recorded in the error_log. We achieve this with the following StatisticsController override in override/controllers/front/StatisticsController.php
`
class StatisticsController extends StatisticsControllerCore
{
    public function display() {

        if(!$this->template) {
            return null;
        }
        parent::display();
    }
}
`